clients/python/lorax/types.py

```diff
@@ -88,7 +88,7 @@ def valid_seed(cls, v):
 
     @validator("temperature")
     def valid_temp(cls, v):
-        if v is not None and v <= 0:
+        if v is not None and v < 0:
```

Comment:
Nit: please change validation error to read "`temperature` must be non-negative".